client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-layout.tsx

```diff
@@ -72,15 +72,15 @@ const PatternLayout = ( {
 							</Button>
 						</li>
 					) }
-					<AsyncLoad require="./animate-list" featureName="domMax">
+					<AsyncLoad require="./animate-list" featureName="domMax" placeholder={ <div /> }>
```

Comment:
Thanks for helping correct this, `placeholder` is similar to `Suspense fallback` right?

Reply:
It is shown while the component is loading. I just had to remove the class to hide it because we don't use it.